index.js

```diff
@@ -2,8 +2,7 @@
 
 module.exports = YahooFantasy;
 
-var OAuth = require('oauth').OAuth,
-  https = require('https'),
+var request = require('request'),
```

Comment:
I think that Yahoo didn't submit OAuth 2.0 for Fantasy API **officially** because of the problem to query the public data. We have three choices,

1. Keep current OAuth 1.0 in yfsapi, and discard this pull request.
2. Merge this pull request which update yfsapi from OAuth 1.0 to OAuth 2.0, and use always the user authentication for all APIs.
3. Mix OAuth 1.0 for public data and OAuth 2.0 for private data. (So complicated..)

What is your opinion? If you select `#1`, I will maintain new [yfsapi-without-auth](https://github.com/githubsmilo/yfsapi-without-auth) module seperately.